### PR TITLE
influxctl 2.3.1 release notes

### PR DESCRIPTION
**Pull request by sanderson**
[x] Rebased/mergeable

### CODE REVIEW

**Review by powersj — APPROVED** [review]

File: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ menu:
...
+## v2.3.1 {date="2023-11-15"}
+
+### Bug fixes
+
+- Include error message description in device authorization polling errors.
+- Do not save tokens when loading from file.
+- Update header comments in the example `config.toml`.
+
+### Dependency updates
+- Update `github.com/golang-jwt/jwt/v5`` from 5.0.0 to 5.1.0.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md:23]:
```suggestion
- Update `github.com/golang-jwt/jwt/v5` from 5.0.0 to 5.1.0.
```
(VSC...am I right?)

File: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md [hunk 2]
```diff
@@ -11,6 +11,20 @@ menu:
...
+- Update `golang.org/x/oauth2`` from 0.13.0 to 0.14.0.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md:24]:
```suggestion
- Update `golang.org/x/oauth2` from 0.13.0 to 0.14.0.
```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ weight: 202
...
+## v2.3.1 {date="2023-11-15"}
+
+### Bug fixes
+
+- Include error message description in device authorization polling errors.
+- Do not save tokens when loading from file.
+- Update header comments in the example `config.toml`.
+
+### Dependency updates
+- Update `github.com/golang-jwt/jwt/v5`` from 5.0.0 to 5.1.0.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:24]:
```suggestion
- Update `github.com/golang-jwt/jwt/v5` from 5.0.0 to 5.1.0.
```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 4]
```diff
@@ -12,6 +12,20 @@ weight: 202
...
+- Update `golang.org/x/oauth2`` from 0.13.0 to 0.14.0.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:25]:
```suggestion
- Update `golang.org/x/oauth2` from 0.13.0 to 0.14.0.
```

File: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md [hunk 5]
```diff
@@ -11,6 +11,20 @@ menu:
...
 
+## v2.3.1 {date="2023-11-15"}
+
+### Bug fixes
+
+- Include error message description in device authorization polling errors.
+- Do not save tokens when loading from file.
+- Update header comments in the example `config.toml`.
+
+### Dependency updates
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md:22]:
```suggestion
### Dependency updates

```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 6]
```diff
@@ -12,6 +12,20 @@ weight: 202
...
 
+## v2.3.1 {date="2023-11-15"}
+
+### Bug fixes
+
+- Include error message description in device authorization polling errors.
+- Do not save tokens when loading from file.
+- Update header comments in the example `config.toml`.
+
+### Dependency updates
```
Comment (sanderson) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:23]:
```suggestion
### Dependency updates

```

**Pull request merged**